### DISCUSSION TITLE
Trouble with NES Carts

### DISCUSSION THREAD

**Discussion by PermafrostFE** [sanni/cartreader#187] [Need help]
I haven't had any success dumping NES carts.  For reference I bought a pre-assembled unit rather than building it myself and was using the NES to SNES adapter that came with it.  I've had it for a little over a year and hadn't tried using it with NES games until now.  I downloaded a hex editor to see if the header was the issue, and the CART.nes is always empty with FF in most every address with a few FBs scattered around.  I tried updating the firmware on the reader, and that seems to have gone successfully as sanni's logo now appears on boot up when it hadn't before, so I'm assuming I did the update correctly.  But it's not giving me any better results dumping NES games.

Now I'm wondering if it's not working because my adapter's board is too thick.  I noticed a warning about being sure to select the correct board thickness in the instructions for making the adapter.  And when I compared my adapter to one of my SNES boards, it is quite a bit thicker.  It can go into the reader's SNES slot, but it's very tight. Would the adapter being too thick cause a bad connection? I double checked the SNES slot with an SNES game, and it's working fine. 

I'm trying to dump Final Fantasy, and I have the reader set to:
Mapper 1
PRG 256
CHR 0

Edit: I got it working. I think it was because I had SRAM set to 0 because I didn't think I needed to dump it.  I would just delete this post, but I don't see any what to do it.

**Comment by jon-nielsen** [comment]
I think you're right that setting all parameters correctly matters, even if you're only dumping PRG for instance. So you could add that detail to documentation instead of deleting the post. I just had the same experience - and looking at how complicated the NES mapper area is, there's probably a reason :-)
